src/java/org/apache/cassandra/tools/Util.java

```diff
@@ -54,6 +54,7 @@
 @SuppressWarnings("serial")
 public final class Util
 {
+    public static final String ALLOW_TOOL_REINIT = "ALLOW_TOOL_REINIT"; // Necessary for testing
```

Comment:
Two nits:
 - the name comes across as very "public", could it be something more obviously test-ish? like [here|https://github.com/apache/cassandra/blob/trunk/src/java/org/apache/cassandra/config/CassandraRelevantProperties.java#L150-L152]
- and should the property be defined in `CassandraRelevantProperties`? (otherwise can be a package protected constant?)

Reply:
Used a slightly diff name. Hope it's ok.

Reply:
Yes yes ofc, we are but bike-shedding.